### PR TITLE
Update Screenshots

### PR DESCRIPTION
**Pull request by sampoearas**
- Updated screenshots to v12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,27 +12,25 @@ This project lists users but it would be very easy to adapt it for any other ite
...
+[v1.0.2](https://github.com/ArasLabs/visual-user-listing/releases/tag/v1.0.2) | Screenshots updated for v12 
 [v1.0.1](https://github.com/ArasLabs/visual-user-listing/releases/tag/v1.0.1) | Tested 11.0 SP12, SP15. Tested on Edge, Internet Explorer, Firefox 60 ESR, Chrome.
 [v1.0.0](https://github.com/ArasLabs/visual-user-listing/releases/tag/v1.0.0) | First release. Tested on Internet Explorer 11, Firefox 38 ESR, Chrome. 
-> Though built and tested using Aras 11.0 SP9, this project should function in older releases of Aras 11.0 and Aras 10.0.
-
 #### Supported Aras Versions
 
 Project | Aras
 --------|------
+[v1.0.2](https://github.com/ArasLabs/visual-user-listing/releases/tag/v1.0.2) | 11.9+, 12.0+
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
[v1.0.2](https://github.com/ArasLabs/visual-user-listing/releases/tag/v1.0.2) | 11.0 SP9+, 12.0+ 
```

File: README.md [hunk 2]
```diff
@@ -12,27 +12,25 @@ This project lists users but it would be very easy to adapt it for any other ite
...
-
 ## Installation
 
 #### Important!
 **Always back up your code tree and database before applying an import package or code tree patch!**
 
 ### Pre-requisites
 
-1. Aras Innovator installed (version 11.0 SPx preferred)
+1. Aras Innovator installed (version 12.0 preferred)
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
1. Aras Innovator installed (see "Supported Aras Versions" above)
```

**Review by EliJDonahue — CHANGES_REQUESTED** [review]
Just a note in addition to the requested changes - there's no reason you *have to* change image names when you update them.

**Comment by sampoearas** [comment]
Suggestions accepted.

**Review by EliJDonahue — APPROVED** [review]

**Pull request merged**